src/positiondetector/SimpleThreshold.h

```diff
@@ -54,18 +54,20 @@ class SimpleThreshold : public PositionDetector {
     void set_max_object_area(double value) { max_object_area_ = value; }
     void set_erode_size(int erode_px);
     void set_dilate_size(int dilate_px);
+    void set_mincomp_size(int mincomp_val);
```

Comment:
Don't need this member function. Those were only needed for the erode and dilate options because I had to set up a little smoothing kernel each time the value is changed.